packages/ai/core/prompt/convert-to-language-model-prompt.ts

```diff
@@ -273,7 +273,7 @@ function convertPartToLanguageModelPart(
 
   let mimeType: string | undefined = part.mimeType;
   let data: DataContent | URL;
-  let content: URL | ArrayBuffer | string;
+  let content: DataContent | URL | string;
```

Comment:
Why was this change necessary?

Reply:
typescript now enforces better typechecking, so this was now a compile error